extensions/interactions/DragAndDropSortInput/directives/oppia-interactive-drag-and-drop-sort-input.component.ts

```diff
@@ -146,6 +171,48 @@ export class InteractiveDragAndDropSortInputComponent implements OnInit {
     moveItemInArray(
       this.singleItemInSamePositionArray,
       event.previousIndex, event.currentIndex);
+    this.activeItem = event.currentIndex;
+    this.setFocus();
+  }
+
+  handleKeyDown(event: KeyboardEvent, currentIndex: number): void {
```

Comment:
@Patel-Muhammad I think you would have to write frontend tests for this function.

Reply:
@Lawful2002 done